Add viral characteristics of CoV to introduction (#136)
greenelab/covid19-review: content/02.introduction.md

In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### About Coronaviruses

#### Classification:
Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses. Several members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species.

#### Virion, morphology, structure:
The virion is spherical with a diameter ranging between 100 to 160nm. The virion has a lipid envelope with spike (S) glycoproteins on the surface creating the distinctive "crown" shape, from which the virus was named. 

#### Genome structure and replication:

> The genome is a single-stranded non-segmented positive RNA (ssRNA+) that is 27 to 32kb in length. The genome is comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. There are additional accessory genes that are present depending on the strain. The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except \gamma coronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA. 

Reviewer: I think this is info about coronaviruses generally and not SARS-CoV-2 in particular, but perhaps make that more explicit?

Reviewer 2: We can cite this accession id and mention that the Wuhan Hu1 strain has 29kb (29,903 bp) genome. https://www.ncbi.nlm.nih.gov/nuccore/MN908947.

Author: Thats a very good point. Yes, the text was meant to give general CoV background. But I think adding was is known about SARS-CoV2 would be helpful.

> Replication initiates with the binding of the virus to host cell via the S protein. The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).

Reviewer: ```suggestion
Replication initiates with the binding of the virus to a host cell via the S protein. 
The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. 
The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
```
If you can reformat to one sentence per line, we're finding that makes it easier to track what changes!

Reviewer 2: Like this?

The replication initiates with the virus binding to the host receptor (ACE2) via the S protein, followed by fusion of viral membrane with host cell membrane to form endosomes. The release of the viral genome into the host cytoplasm initiates the replicase gene to translate and assemble into a replicase complex that synthesizes dsRNA from its +ssRNA.

Reviewer: Thanks for looking at this @raghuyennamalli ! 
Here is a description for how to suggest additions/deletions/changes that will result in GitHub logging your work on the text:
https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review
I usually use that for any edits I'm doing to the text, and then use comments to ask questions, etc.

In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### About Coronaviruses

#### Classification:
Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses. 
Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2.

#### Virion, morphology, structure:
Coronavirus virions are spherical with diameters ranging between 100 to 160nm. 
The virion has a lipid envelope with spike (S) glycoproteins on the surface creating a distinctive "crown" shape, for which the family of viruses was named. 

#### Genome structure and replication:
Coronavirus genomes are single-stranded non-segmented positive RNA (ssRNA+) that are 27 to 32kb in length. 

> The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp in length [@doi:10.1016/S0140-6736(20)30251-8].

Reviewer: ```suggestion
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp in length [@doi:10/ggjr43].
```

Coronavirus genomes are comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins. 
There are additional accessory genes that are present depending on the strain. 
The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except Gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA. 

Replication initiates with the binding of the virus to a host cell via the S protein. 
The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm. 
The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes.  

> 

Reviewer: ```suggestion
```

#### Transmission:
In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes). 
While there does not appear to be experimental evidence to conclude which mode of transmission different coronaviruses use [@doi:10.1038/s41368-020-0075-9], it is suggested that the coronavirus person-to-person transmission is droplet-based using simulations [@doi:10.1056/NEJMoa032867] and trace studies[@doi:10.3390/ijerph15122757].

> References:

Reviewer: I didn't notice these! Is there somewhere they could go inline?

Author: yes I'll update this
